documentation/introduction/tutorial-get-started-first-part.asciidoc

```diff
@@ -2,87 +2,103 @@
-around with it.
+Use the result as a starting point for more Flow experiments, such as using add-ons, adding a custom theme, adding new views, or optimizing for mobile. 
 
-[[flow.tutorial.overview.tools]]
-=== Installing the Development Tools
+The tutorial should take 20 - 60 minutes to complete. 
 
-The tutorial uses Java 8, so please ensure that you have an up-to-date JDK 8
-installed. Most Linux distributions can use package managers to install JDK8.
-Windows and Mac users should download it from http://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle's Java SE site].
-Also, make sure you have the latest version of your IDE. Eclipse is available in
-various packages; be sure to download the *Eclipse IDE for Java EE Developers*
-from http://www.eclipse.org/downloads/[eclipse.org].
+As an alternative to coding from scratch, you can 
+https://github.com/vaadin/flow-and-components-documentation/tree/master/tutorial-getting-started[download the final application] and experiment with it.
 
-[[flow.tutorial.overview.projectbase]]
-=== Starting with the Project Base
 
-As the starting point for the application, we use a Vaadin Application Starter, called Project Base for Vaadin 10. Project Bases are project stubs that have dependencies set up, and may include some example code. The steps of creating a Vaadin Application are as follows:
 
-. Start by pointing your browser at https://vaadin.com/start[Vaadin Start]
+=== Installing the Development Tools 
 
-. Select the Project Base for V10 by clicking it
+* *JDK 8*: The tutorial uses Java 8. Make sure that you have JDK 8 installed and that it is up to date.
+** Most Linux distributions can use package managers to install JDK 8.
+** Windows and Mac users need to download JDK 8 from http://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle's Java SE Downloads].
 
-. For [guilabel]#Group ID#::Give `com.vaadin.starter.skeleton`
+* *IDE*: The tutorial uses Eclipse, but you can use any IDE. Make sure you have the latest version installed. 
+** To download Eclipse, select *Eclipse IDE for Java EE Developers* from the available packages at https://www.eclipse.org/downloads/packages/[eclipse.org].
 
-. For [guilabel]#App Name#::Give `my-app`
 
-. Click [guilabel]#download# button to download the preconfigured project stub as a ZIP file. (If the download button is not active, you should log in to download the starter).
 
-. The name of the downloaded ZIP package should be `my_app.zip`. Locate it from your hard drive and extract it to a folder of your preference.
+=== Installing the Project Base
 
-. Fire up Eclipse, and from the file menu, select  import. From the select window choose Existing Maven Project from Maven section. Select the [guilabel]#next# button. Select the [guilabel]#Browse# button to choose your project and finally click the [guilabel]#finish# button.
+The tutorial uses the *Project Base* starter pack for Vaadin 12. 
```

Comment:
We should probably avoid using explicit version numbers here, otherwise we'll need to update docs on each Vaadin release, which are happening quite often now due to the release train model.

@pleku Do you know if we have some variable that we could use? I guess in this case we could though just leave the version number out.